.grit/workflows/stainless/workflow.ts

```diff
@@ -38,11 +57,7 @@ function findListNestingModeBlockTypes(schema: z.infer<typeof CloudflareSchema>)
   for (const [resourceName, resourceSchema] of Object.entries(resourceSchemas)) {
     const blockTypes = resourceSchema.block.block_types;
     if (blockTypes) {
-      for (const [attributeName, blockType] of Object.entries(blockTypes)) {
-        if (blockType.nesting_mode === "list") {
-          results.push({ resource: resourceName, attribute: attributeName });
-        }
-      }
+      findRecursiveBlockTypes(results, resourceName, blockTypes);
```

Comment:
i think we need to split this up a little more from our slack chat. where the definition includes a `"nesting_mode": "single"` we need it to be `foo = { .. }` but when the definition includes `"nesting_mode": "list"` we want to to `foo = [{ .. }]`.

old 

```hcl
resource "cloudflare_load_balancer_pool" "example" {
  account_id = "f037e56e89293a057740de681ac9abbe"
  name       = "example-pool"
  origins {
    name    = "example-1"
    address = "192.0.2.1"
    enabled = false
    header {
      header = "Host"
      values = ["example-1"]
    }
  }
  origins {
    name    = "example-2"
    address = "192.0.2.2"
    header {
      header = "Host"
      values = ["example-2"]
    }
  }
  latitude           = 55
  longitude          = -12
  description        = "example load balancer pool"
  enabled            = false
  minimum_origins    = 1
  notification_email = "someone@example.com"
  load_shedding {
    default_percent = 55
    default_policy  = "random"
    session_percent = 12
    session_policy  = "hash"
  }
  origin_steering {
    policy = "random"
  }
}

```

new

```hcl
resource "cloudflare_load_balancer_pool" "example" {
  account_id = "f037e56e89293a057740de681ac9abbe"
  name       = "example-pool"
  
  origins = [{
    name    = "example-1"
    address = "192.0.2.1"
    enabled = false
    header {
      header = "Host"
      values = ["example-1"]
    }
  },
  {
    name    = "example-2"
    address = "192.0.2.2"
    header {
      header = "Host"
      values = ["example-2"]
    }
  }]
  
  latitude           = 55
  longitude          = -12
  description        = "example load balancer pool"
  enabled            = false
  minimum_origins    = 1
  notification_email = "someone@example.com"
  
  load_shedding = {
    default_percent = 55
    default_policy  = "random"
    session_percent = 12
    session_policy  = "hash"
  }
  origin_steering = {
    policy = "random"
  }
}
```

Reply:
844e4ed looks to get this sorted. nice one!

Reply:
Thanks @jacobbednarz, I added a side condition to handle the nested map types. Hopefully this is getting close now.